app/controllers/admin/admin_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::AdminController < Admin::BaseController
```

Comment:
was this not supposed to move to users controller?

Reply:
Moved.